.github/workflows/ci.yml

```
@@ -14,82 +14,134 @@ on:
       - name: Setup .NET 9.0
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: ${{ env.DOTNET_VERSION_2 }}
+          dotnet-version: '9.0.x'
 
       - name: Install dotnet-format
         run: dotnet tool install --global dotnet-format
-      - name: Run formatter for lib
+
+      - name: format lib
         run: dotnet format ${{ env.PROJECT_PATH }} --verify-no-changes 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)